### PR TITLE
Zachi branch

### PR DESCRIPTION
**Pull request by mzachar6**
Issue #14 display list of playgrounds - done

### CODE REVIEW

**Comment by buksy90** [comment]
you didnt met two of three AC ( acceptance criteria) from task. 
your branch also needs to be rebased to fix the conflicts.

**Comment by buksy90** [comment]
I just run your branch and it looks good with card. So I will leave this up to you to decide whether we will use card or table

File: src/components/App.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ import MainMenu from "./MainMenu";
 
 const Home = lazy(() => import('../pages/Home'));
 const Schedule = lazy(() => import('../pages/Schedule'));
+const PlayGroundList = lazy(() => import('../pages/PlaygroundList'));
```
[the remark this answers is not in the document]
Comment (mzachar6) [reply src/components/App.js]: The format of variable name has been aligned with PlaygroundList file

File: src/components/App.js [hunk 2]
```diff
@@ -23,6 +24,8 @@ class App extends Component {
             <Switch>
               <Route exact path="/" component={props => <Home {...props}/>}/>
               <Route path="/schedule" component={props => <Schedule {...props}/>}/>
+              <Route path="/playground" component={props => <PlayGroundList heading="Ihriská"
+               listTitle="Zoznamy ihrísk"/>}/>
```
[the remark this answers is not in the document]
Comment (mzachar6) [reply src/components/App.js]: I've fixed it :)

**Comment by buksy90** [comment]
good fixes, but you shouldnt have merge master branch INTO your branch. Correct way is to always merge feature branches INTO master, not vice versa. If you want to apply changes from master into your branch, you have to rebase your branch against the master. 
Since merging into master branch is denied in this repository, you have to create pull request (as you already did) and when pull request passes code review you can merge your branch via github pull request.

You have to either revert your merge commit now or (maybe easier way) create new branch and cherry pick your commits into new branch.

**Comment by buksy90** [comment]
By the way, 
eslint is reporting an issue on line 7 of playground.js. you cannot have an empty "href" attribute. Set it to "#playground" for now to fix the eslint.

**Comment by buksy90** [comment]
Im closing this one since changes have been made in another PR

**Pull request closed without merging**